### PR TITLE
[StyleCop] Fix all warnings in Japanese\TimePeriodParser file

### PR DESCRIPTION
**Pull request by JuanAr**
- Classes were separated into different files.
- Reordered methods.
- Added spaces and trailing commas when needed.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimePeriodFunctions.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
+﻿using System;
```
Comment (guom08) [inline .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimePeriodFunctions.cs]: The same, I'm not quite sure whether we should put this file under "Parsers" folder or not. @tellarin

Comment (tellarin) [reply .NET/Microsoft.Recognizers.Text.DateTime/Japanese/Parsers/TimePeriodFunctions.cs]: I agree with Mu. They seem generic and not Japanese-specific. Maybe in /Utilities?

**Comment by JuanAr** [comment]
@tellarin @guom08 We'll close this PR and do the class refactor: unify Chinese and Japanese as they both use identical public classes and move them to the Utilities folder. We'll fix the StyleCop warnings as part of this new PR.

**Comment by tellarin** [comment]
Thanks!

**Pull request closed without merging**